model/src/main/java/com/javahelp/model/token/Token.java

```diff
@@ -18,17 +17,17 @@ public class Token {
     /**
      * Expiry date of the token.
      */
-    private LocalDate expiry;
+    private Instant expiry;
 
     /**
      * Issued date of the token.
      */
-    private LocalDate issued;
+    private Instant issued;
```

Comment:
What is an instant?

Reply:
Um its like a LocalDate but not connected to any timezone

Reply:
or i mean a LocalDate isn't connected to a timezone but it does assume that there's some associated timezone implicitly

Reply:
Also LocalDate is just for dates, and doesn't do times. But basically, LocalDate represents like a day on a Calendar. But the problem is, we may have people in multiple timezones. We could still get around that (using LocalDateTime to allow for tracking time) by converting all times to a specific time zone before we interact with them, but it's easier to use something like an Instant, which instead of representing a calendar date, represents a specific moment in time, and is the same across timezones.